### PR TITLE
#12: Invalid syntax, when enforce variable is default (empty string)

### LINKED ISSUE #12: Invalid syntax, when enforce variable is default (empty string)
The logic assumes var.enforce to be one of
- true
- false
- "" [(default)](https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/variables.tf#L34)

But we can't use `not` operator with empty string.

Offending lines:
- [enforced = "${!var.enforce}"](https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/boolean_constraints.tf#L69)
- [enforced = "${!var.enforce}"](https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/boolean_constraints.tf#L55)

```
[16:09][user@host:~]$ /home/data/Soft/terraform_11/terraform -v
Terraform v0.11.14

Your version of Terraform is out of date! The latest version
is 0.12.5. You can update by downloading from www.terraform.io/downloads.html
[16:09][mulder@my:~]$ /home/data/Soft/terraform_11/terraform console
> ! ""
__builtin_StringToBool: strconv.ParseBool: parsing "": invalid syntax in:

${! ""}
> ! true
false
> ! false
true
>  
```

This bug is not covered by tests or examples, however it's likely fixed (to be confirmed!) in [TF 0.12 PR](https://github.com/terraform-google-modules/terraform-google-org-policy/pull/11)

### PR DESCRIPTION
**Pull request by nick4fake**
Fixes #12

Updated var.enforce default to "true"

### CODE REVIEW

**Comment by nick4fake** [comment]
@averbuks This PR was marked with "work in progress", but thank you for review. I'll push updated PR soon

File: test/integration/list_constraints/launch.sh [hunk 1]
```diff
@@ -62,7 +62,7 @@ module "org-policy-list-project" {
   policy_for  = "project"
   constraint  = "$PROJECT_CONSTRAINT_DENY_ALL"
   project_id  = "$PROJECT_ID"
-  enforce     = "true"
+  #enforce     = "true"
```
Comment (aaron-lane) [inline test/integration/list_constraints/launch.sh]: Commented-out code should be removed.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -38,8 +38,8 @@ To control module's behavior, change variables' values regarding the following:
   - `enforce`: if "true" the policy is enforced at the root, if "false" the policy is not enforced at the root. (default `true`)
```
Comment (aaron-lane) [inline README.md]:
```suggestion
  - `enforce`: if `true` or `null` then the policy is enforced at the root; if `false` then policy is not enforced at the root. (default `null`)
```

File: README.md [hunk 3]
```diff
@@ -38,8 +38,8 @@ To control module's behavior, change variables' values regarding the following:
...
 - List policies (with `policy_type: "list"`) can set **one of** the following variables. Only one may be set.
   - `enforce`: if "true" policy will deny all, if "false" policy will allow all (default `true`)
```
Comment (aaron-lane) [inline README.md]:
```suggestion
  - `enforce`: if `true` or `null` then policy will deny all; if `false` then policy will allow all (default `null`)
```

File: test/integration/boolean_constraints/launch.sh [hunk 4]
```diff
@@ -58,7 +58,7 @@ module "org-policy-boolean-project" {
   policy_for  = "project"
   constraint  = "$PROJECT_CONSTRAINT"
   project_id  = "$PROJECT_ID"
-  enforce     = "true"
+  #enforce     = "true"
```
Comment (aaron-lane) [inline test/integration/boolean_constraints/launch.sh]: Commented-out code should be removed.

```suggestion
```

**Pull request merged**